CHANGELOG.md

```diff
@@ -41,6 +41,8 @@ different versioning scheme, following the Haskell community's
   `GetValueToNameMap`.
 * C++ codegen now applies the `--export-attribute` to the `ToString`,
   `FromString`, `ToEnum` and `FromEnum` functions.
+* C++ codegen now generates `allocator_type` typedef for a struct when `--allocator`
```

Comment:
Minor: "*an* `allocator_type` typedef for a struct when *the* `--allocator` *flag* is passed to `gbc`..."